site/docs/2.2.3/configuration.html

```diff
@@ -134,10 +134,33 @@ <h1 class="title">Spark Configuration</h1>
           <li><a href="#runtime-environment" id="markdown-toc-runtime-environment">Runtime Environment</a></li>
           <li><a href="#shuffle-behavior" id="markdown-toc-shuffle-behavior">Shuffle Behavior</a></li>
           <li><a href="#spark-ui" id="markdown-toc-spark-ui">Spark UI</a></li>
+          <li><a href="#compression-and-serialization" id="markdown-toc-compression-and-serialization">Compression and Serialization</a></li>
```

Comment:
Is this part of the 'hotfix' or new additional docs from later in the branch?
It's fine by me too, just checking.

Reply:
It's the part of this hot fix. The broken tag makes the rest of doc is not generated correctly.